adc/messages.go

```diff
@@ -277,6 +279,26 @@ func (UserMod) Cmd() MsgType {
 	return MsgType{'I', 'N', 'F'}
 }
 
+var _ Message = GetPassword{}
+
+type GetPassword struct {
+	Data []byte
```

Comment:
Please add tests - I'm not sure it will read it automatically. It should have at least an `adc:"#"` tag for it to work.